content/en/docs/concepts/scheduling-eviction/dynamic-resource-allocation.md

```diff
@@ -0,0 +1,161 @@
+---
+reviewers:
+- klueska
+- pohly
+title: Dynamic Resource Allocation
+content_type: concept
+weight: 65
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.26" state="alpha" >}}
+
+Dynamic resource allocation is a new API for requesting and sharing resources
+between pods and containers inside a pod. Third-party resource drivers are
+responsible for tracking and allocating resources. The different kinds of
+resources support arbitrary parameters for defining requirements and
+initialization.
+
+## {{% heading "prerequisites" %}}
+
+Kubernetes v{{< skew currentVersion >}} includes cluster-level API support for
+dynamic resource allocation, but it [needs to be
+enabled](#enabling-dynamic-resource-allocation) explicitly.  You also must
+install a resource driver for specific resources that are meant to be managed
+using this API.  If you are not running Kubernetes v{{< skew currentVersion>}},
+check the documentation for that version of Kubernetes.
+
```

Comment:
Should we point them at the e2e test here for now to see what an example resource driver looks like (and in the future point to the `dra-example´driver` repo we are in the process of creating)?

Reply:
I prefer to do a future update of this page once we have a readily available example driver. 

The test driver feels a bit too "rough" to be useful here. In the feature blog post we can link to it.